projects/wave-core/src/lib/operators/dialogs/data-repository/vector/vector-source-dataset.component.ts

```diff
@@ -96,6 +96,29 @@ export class VectorSourceDatasetComponent implements OnInit {
         });
         this.projectService.addLayer(l);
     }
+    ngOnChanges(changes: SimpleChanges) {
+        for (const key in changes) {
+            if (changes.hasOwnProperty(key)) {
+                switch (key) {
+                    // check if there is any time-validity start/end data. If there is start/end data show the column of this data.
+                    case 'dataset': {
+                        this.dataset.rasterLayer.forEach((element) => {
+                            if (element.time_start.isValid()) {
```

Comment:
What happens if `time_start` is undefined?

Reply:
fixed in 1a0b23afc2302b363ceb11f273d325f130d8539d